### DISCUSSION TITLE
Change the text of buttons in edit modal

### DISCUSSION THREAD

**Discussion by hussain7006** [KevinVandy/material-react-table#1148] [Q&A]
### material-react-table version

v2

### react & react-dom versions

v18.2

### Describe the bug and the steps to reproduce it

Change the text of buttons in the edit modal

### Minimal, Reproducible Example - (Optional, but Recommended)

Change the text of buttons in edit modal

### Screenshots or Videos (Optional)

Change the text of buttons in edit modal

### Do you intend to try to help solve this bug with your own PR?

No, because I do not know how

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by KevinVandy** [comment]
Please ask question in Discussions. This is not a bug.

**Comment by KevinVandy** [comment]
You're probably looking for https://www.material-react-table.com/docs/guides/localization or the `renderEditRowDialogContent` table option.
